es_ES/faq.md

```diff
@@ -0,0 +1,625 @@
+
+El sistema de interoperabilidad y plugin de Flutter está diseñado para permitir
+desarrolladores para acceder a las nuevas funciones y capacidades del sistema operativo móvil inmediatamente. Los desarrolladores no tienen que esperar al equipo de Flutter para exponer la nueva capacidad del sistema operativo móvil.
+
+### Qué sistemas operativos puedo usar para construir una aplicación de Flutter?
+
+Flutter admite el desarrollo en Linux, Mac y Windows.
+
+### En qué idioma está escrito Flutter?
+
+Observamos muchos lenguajes y tiempos de ejecución, y finalmente adoptamos a Dart para el framwork y los widgets. El framework de gráficos subyacente y la máquina virtual de Dart se implementan en C/C++.
+
+### Por qué Flutter eligió usar Dart?
+
+Flutter utilizó cuatro dimensiones principales para la evaluación, y consideró las necesidades de autores de framework, desarrolladores y usuarios finales. Encontramos algunos lenguajes encontrados algunos requisitos, pero Dart obtuvo una puntuación alta en todas nuestras dimensiones de evaluación y cumplió todos nuestros requisitos y criterios.
+
+Los tiempos de ejecución Dart y los compiladores admiten la combinación de dos características críticas para Flutter: un ciclo de desarrollo rápido basado en JIT que permite el cambio de forma y hot reload con estado en un idioma con tipado, más un compilador AOT que emite un código ARM eficiente para un arranque rápido y un rendimiento predecible de implementaciones de producción.
+
+Además, tenemos la oportunidad de trabajar estrechamente con la comunidad Dart,
+que está invirtiendo activamente recursos para mejorar Dart para usar en Flutter. por
+ejemplo, cuando adoptamos Dart, el lenguaje no tenía una AOT toolchain para producir binarios nativos, que es fundamental para lograr predecible, de alto rendimiento, pero ahora el lenguaje lo hace porque el equipo de Dart lo construyó para Flutter. Del mismo modo, la Dart VM se ha optimizado previamente para rendimiento pero el equipo ahora está optimizando la VM para la latencia, que es más
+importante para la carga de trabajo de Flutter.
+
+Dart tiene una alta puntuación para nosotros en los siguientes criterios principales:
+
+* _Productividad del desarrollador_. Una de las principales propuestas de valor de Flutter es que ahorra recursos de ingeniería al permitir que los desarrolladores creen aplicaciones tanto para iOS como para Android con la misma base de código. Usar un lenguaje altamente productivo acelera aún más a los desarrolladores y hace que Flutter sea más atractivo. Esto fue muy importante tanto para nuestro equipo de framework como para nuestros desarrolladores. La mayoría de Flutter está construido en el mismo idioma que le damos a nuestros usuarios, por lo que necesitamos seguir siendo productivos en las líneas de código de 100k, sin sacrificar accesibilidad o legibilidad del framework y widgets para nuestros desarrolladores.

```

Comment:
so we need to stay productive at 100k’s lines of code
necesitamos seguir siendo productivos en sus mas de 100 mil líneas de código, sin...